tsconfig.json

```diff
@@ -17,7 +17,8 @@
       "DOM",
       "ES5",
       "ES6",
-      "ES7"
+      "ES7",
+      "ES2021.String"
```

Comment:
Is "ES2021.String" needed?

Reply:
Ok I see it's need for that [`replaceAll`](https://github.com/jaanonim/obsidian-youversion-linker/pull/32/files#diff-f9f01c50435a34208cf5fcdf172fd8c4c324da6cc09522cd9a695e4be9a648e1R120) but I don't think that `replaceAll` will stay there as it throws errors.